commcare/src/test/java/org/motechproject/commcare/it/CommcareTasksIntegrationBundleIT.java

```diff
@@ -313,8 +314,8 @@ private HttpResponse sendMockForm() throws IOException, InterruptedException {
         return getHttpClient().execute(httpPost);
     }
 
-    private HttpResponse createConfiguration(Config config) throws IOException, InterruptedException {
-        HttpPost httpPost = new HttpPost(String.format("http://localhost:%d/commcare/configs", PORT));
+    private HttpResponse createConfiguration(Config config, String oldName) throws IOException, InterruptedException {
+        HttpPost httpPost = new HttpPost(String.format("http://localhost:%d/commcare/configs?oldName=%s", PORT, oldName));
```

Comment:
why you used this request to create config ? I thought that there are two requests, one for create and second for update config.